### ISSUE TITLE
Multiples styles/theme

### ISSUE THREAD

**Issue by Jeconias** [Jeconias/calendarjs#9]
This issue is intended to enable and drive the creation of various styles for the calendar using css.

If you want to create a style:

- Create folder in stylesheet directory with the name of style and add preview.

**Comment by kevinsunny1996** [comment]
Hey @Jeconias  does this issue need few more styles? If yes...Please assign this one to me

**Comment by Jeconias** [comment]
> Hey @Jeconias does this issue need few more styles? If yes...Please assign this one to me

Yes. The idea is to have many styles and allow to exchange between them easily using some flag.

**Comment by juanmagutierrez** [comment]
Maybe the easiest and more flexible solution is to use two css files. If you import them in the right order the second one can override the default theme, and you only have to write the properties that changes from the default theme.

For example:
https://stackoverflow.com/questions/5902858/order-of-prioritization-when-using-multiple-contradictory-css-files

**Comment by Jeconias** [comment]
> Maybe the easiest and more flexible solution is to use two css files. If you import them in the right order the second one can override the default theme, and you only have to write the properties that changes from the default theme.
> 
> For example:
> https://stackoverflow.com/questions/5902858/order-of-prioritization-when-using-multiple-contradictory-css-files

This is good. should work well.
